### ISSUE TITLE
Space bar action - "Start Next" playing two slots in the same time

### ISSUE THREAD

**Issue by radiocafero** [ElvishArtisan/rivendell#541]
Hi Fred!
Either I don't realize what's going on or it's a bug.
From two versions ago...
I hope this release will solve the problem if it really is a problem.
If you check "start next" from spacebar keyboard, in rdadmin, and you want to start next with space, then it works well till play something from carts.
After this action same space keyboard starts two tracks in the same time.
Rivendell V3.3.0, Centos7.

**Comment by ElvishArtisan** [comment]
I hope this release will solve the problem if it really is a problem.
> If you check "start next" from spacebar keyboard, in rdadmin, and you want to start next with space, then it works well till play something from carts.
After this action same space keyboard starts two tracks in the same time.
Rivendell V3.3.0, Centos7.

Tested on v3.3.0 as follows:
1) Start rdadmin(1), touch 'Manage Hosts', select desired host in the 'Rivendell Host List' dialog, touch 'Edit', then touch 'RDAirPlay', then set the 'Space Bar Action' to 'Start Next' (lower right hand corner of the dialog). Touch 'OK' to save.

2) Start rdairplay. Load a log. 

3) Tap the space bar. First event begins. 

4) Tap the space bar. Second event starts, first event crossfades and stops.

5) Do a 'Make Next' to an event a dozen or so lines down.

6) Tap the space bar. The designated next event starts. The previously playing event from 4. crossfades and stops.

In short, I cannot reproduce the problem here. That may mean that it's fixed in v3.3.0 (unlikely), or that I'm simply not testing correctly. What exactly does 'play[ing] something from carts' mean?

**Comment by radiocafero** [comment]
then something strange happens here ...
indeed, it does not do for the first time.
I click on a button in the cart and then the it happened.
that disappears only after restarting rdairplay.

**Comment by radiocafero** [comment]
I apologize for the confusion!
cart = sound panel
working with several radios and automation, some colleagues call it cart :-).
Sorry!

**Comment by ElvishArtisan** [comment]
> I apologize for the confusion!
cart = sound panel

Ah, that it! Confirmed here.

> working with several radios and automation, some colleagues call it cart :-).

No worries!

**Comment by ElvishArtisan** [comment]
Fixed in d0c232e.
